src/DIRAC/StorageManagementSystem/Agent/StageMonitorAgent.py

```diff
@@ -93,8 +87,6 @@ def __monitorStorageElementStageRequests(self, storageElement, seReplicaIDs, rep
                 "StageMonitor.__monitorStorageElementStageRequests: No requests to monitor for %s." % storageElement
             )
             return
-        oAccounting = DataOperation()
-        oAccounting.setStartTime()
```

Comment:
Is this lost?

Reply:
This is getting expressed in DataOperationSender

Reply:
not quite true. `setStartTime` was using `utcnow`, you use `time.time`

Reply:
Oops, you are right, will fix it.